runhouse/servers/http/http_server.py

```diff
@@ -554,6 +556,40 @@ async def _get_results_and_logs_generator(fut, run_name, serialization=None):
             for f in open_logfiles:
                 f.close()
 
+    @staticmethod
+    @app.get("/processes")
+    @validate_cluster_access
+    async def get_processes(request: Request):
+        try:
+            processes = await obj_store.aget_all_initialized_servlet_names()
+            return Response(
+                output_type=OutputType.RESULT_SERIALIZED,
+                data=serialize_data(processes, "json"),
+                serialization="json",
+            )
+        except Exception as e:
+            return handle_exception_response(
+                e, traceback.format_exc(), from_http_server=True
+            )
+
+    @staticmethod
+    @app.post("/create_process")
```

Comment:
how come we're exposing it as a "process" to the user, as opposed to a servlet?

Reply:
it's just the wording we decided on that's most clear with @dongreenberg . since each is literally an independent OS prcoess